ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/context/KsqlSecurityContextBinder.java

```diff
@@ -31,9 +34,17 @@
 public class KsqlSecurityContextBinder extends AbstractBinder {
   public KsqlSecurityContextBinder(
       final KsqlConfig ksqlConfig,
-      final KsqlSecurityExtension securityExtension
+      final KsqlSecurityExtension securityExtension,
+      final ServiceContext serviceContext
   ) {
-    KsqlSecurityContextBinderFactory.configure(ksqlConfig, securityExtension);
+    KsqlSecurityContextBinderFactory.configure(ksqlConfig, securityExtension, serviceContext);
+  }
+
+  public static BiFunction<KsqlConfig, KsqlSecurityExtension, Binder> createFactory(
```

Comment:
Is this necessary if we you pass all the require parameters during the construction of the class?

Reply:
Nope, removed.